sympy/series/limits.py

```diff
@@ -162,8 +162,12 @@ def doit(self, **hints):
                                         for m in Mul.make_args(a))
                                     for a in Add.make_args(w)))
                 if all(ok(w) for w in e.as_numer_denom()):
-                    u = Dummy(positive=(z0 is S.Infinity))
-                    inve = e.subs(z, 1/u)
+                    if z0 is S.NegativeInfinity:
+                        u = Dummy(positive=True)
+                        inve = e.subs(z, -1/u)
+                    else:
+                        u = Dummy(positive=(z0 is S.Infinity))
```

Comment:
I think that limit is currently taken on the real line alone. This is implicit in the the docstring

    z0 can be any expression, including oo and -oo.

    For dir="+" (default) it calculates the limit from the right
    (z->z0+) and for dir="-" the limit from the left (z->z0-).  For infinite
    z0 (oo or -oo), the dir argument is determined from the direction
    of the infinity (i.e., dir="-" for oo).

even though 'any expression' would seem to allow complex values.